code/core/src/cli/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from './detect';
+export * from './helpers';
+export * from './angular/helpers';
+export * from './dirs';
+export * from './project_types';
+export * from './NpmOptions';
+export * from './eslintPlugin';
```

Comment:
shared stuff reused in `toolbox` and `create-storybook`

Reply:
Did you consider putting this in `@storybook/core/common`, rather than making a new entrypoint?

Reply:
I guess to not burden any user of `common` with the extra code?

Reply:
But on the other hand, each additional part we'll eventually want to consolidate into actual "public" facing exports..

Would you be open to moving these into the `core/src/common/index` ?

Reply:
@kasperpeulen This comment isn't resolved yet.

Reply:
We can refactor internally later.